Snakefile

```diff
@@ -161,8 +160,9 @@ rule download_local_data:
 rule download_cancerhotspots:
     output: f'{DATA_DIR}/cancerhotspots/cancerhotspots.v2.maf'
     shell: dedent(f'''\
+        mkdir -p {DATA_DIR}/cancerhotspots
         cd {DATA_DIR}/cancerhotspots
-        wget http://download.cbioportal.org/cancerhotspots/cancerhotspots.v2.maf.gz
+        wget https://www.cancerhotspots.org/files/hotspots_v2.xls
         gunzip cancerhotspots.v2.maf.gz
```

Comment:
If we go forward with the hotspots_v2.xls file, then this gunzip step is not needed anymore.